tests/lmeval/test_lmeval.py

```diff
@@ -160,7 +163,11 @@ def _run_lm_eval(self):
             logger.info(
                 f"Comparing {metric}: Expected {expected_val}, Got {actual_val}"
             )
-            assert numpy.isclose(expected_val, actual_val, rtol=0.05)
+            if self.lmeval.use_stderr_atol:
+                std_err = metrics.get(metric.replace(",none", "_stderr,none"))
+                assert numpy.isclose(expected_val, actual_val, atol=std_err)
```

Comment:
Here it seems like you're using the standard error of the quantized model as a dynamic metric for determining absolute tolerance. This may be flawed, as if a model produces garage results, its std_err will be very large, and will therefore be less likely to trigger the threshold.

If anything, we should be using static tolerances determined from the standard error of the unquantized model on this task

> Okay, but you'd have to use the stderr of the base model

Reply:
I updated this to use stderr as absolute tolerance only if provided in metrics. Lmk what you think